c2corg_ui/static/js/card.js

```diff
@@ -31,6 +31,11 @@ app.cardDirective = function($compile, $templateCache) {
       var doc = scope['cardCtrl']['doc'];
       var type = app.utils.getDoctype(doc['type']);
       element.html(template(type));
+      
+      if (type === 'outings') {
```

Comment:
I would prefer we avoid adding doc-specific code.
I have a similar problem with the advanced search => see the ``ng-class`` at https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/partials/advancedsearch.html#L2
By the way I had renamed the "outing" class to "outings" so that we can simply use the document type.